src/styles/_header.scss

```diff
@@ -2,7 +2,7 @@ header {
   background: url('/assets/img/header/red-header-background.jpg');
   background-size: cover;
 
-  div {
+  > div {
```

Comment:
Picking that little guy out saved me like 3 conflicts between the logo and my dropdown images